front-end/components/mathview/Logic/edit.ts

```
@@ -1,6 +1,5 @@
-import { Cursor, Exponent, Fraction, Row, Symbol } from '../Types'
-import { getAdjacentNodes, getIndex, insertNode, moveNode } from './helperFunctions'
-import { createId } from '../MathEditor'
+import { Cursor, Exponent, Fraction, Row, Symbol } from '../core/types'
```

Biome (2.1.2)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)